### ISSUE TITLE
Auto focus number / show keyboard 

### ISSUE THREAD

**Issue by HolgerJeromin** [westnordost/StreetComplete#227]
It would save one click on every quest if I do not have to focus the text input. 
Housenumber, building level quests do have this issue

**Comment by westnordost** [comment]
The number is auto focused. But the keyboard is not shown immediately. This is something I will not change, though, because the keyboard would immediately hide (additionally) 1/3 of the screen.
On click of a quest, the user expects the bottom sheet to not immediately obscure the map but only after he slides it up.

**Comment by westnordost** [comment]
Not sure if this is possible, but perhaps "double clicking" on a quest could immediately expand the quest form to full size + activate keyboard. If you find this is a reasonable idea and would also solve your use case, please create a new ticket for that.

**Comment by HolgerJeromin** [comment]
OK, I retested it. We have really a estate problem. Not seeing the map is not good. You are right 

Providing a narrow custom number pad beneath the number plate does not work on the building level quest. 
:-(

**Comment by matkoniecz** [comment]
>  On click of a quest, the user expects the bottom sheet to not immediately obscure the map but only after he slides it up.

Well, at least one user (me) expects that it would not be necessary to confirm again that I really want to start solving some quest. I am betting that #266 (can't add housenumbers that are not numbers in 0.11 release) is caused not be subtle bug but by user not understanding that he needs to click on the blue area to start number input.

It would be nice to have both full map and focused input from start - but at least for me I prefer the second, even if it completely hides the map. And given that at least in address input part of map will be still visible I think that autofocusing is preferable.
